mypy/fastparse.py

```diff
@@ -43,15 +45,21 @@ def parse(source: Union[str, bytes], fnam: str = None, errors: Errors = None,
     """
     is_stub_file = bool(fnam) and fnam.endswith('.pyi')
     try:
-        ast = typed_ast.parse(source, fnam, 'exec')
+        if pyversion[0] == 3 or is_stub_file:
+            ast = ast35.parse(source, fnam, 'exec')
+        else:
```

Comment:
I'd assert that pyversion[0] == 2 here, or else use >= 3 two lines up (which is what every other pyversion test in mypy uses).